emap-setup/emap_runner/runner.py

```
@@ -188,28 +197,28 @@ def docker(self) -> None:
 
         return None
 
-    def validation(self) -> None:
+    def validation(self) -> ValidationRunner:
         """Run a validation run of EMAP"""
         # allow for hoover not to be defined in global config
-        use_hoover = ("hoover" in self.global_config["repositories"]) and (not self.args.use_only_hl7_reader)
+        use_hoover = ("hoover" in self.global_config["repositories"]) and self.args.use_hoover

```

perhaps this should be

```python
if self.args.use_hoover and "hoover" not in self.global_config["repositories"]:
    raise ValueError("hoover missing from repositories in global config")
```

---

Ah yes, that avoids failing silently if the user asked for hoover